res/mixxx.qrc

```diff
@@ -1,11 +1,7 @@
 <RCC>
     <qresource prefix="/">
         <file>../LICENSE</file>
-      <!-- window icon used in /src/mixxx.cpp and /src/library/scanner/libraryscannerdlg.cpp -->
-      <!-- svg is not accepted so we need to stick with the png here -->
-        <file>images/ic_mixxx_window.png</file>
-      <!-- mouse icon for drag'n'drop actions. won't work with SVG -->
-        <file>images/library/ic_library_drag_and_drop.png</file>
+        <file>images/library/ic_library_drag_and_drop.svg</file>
```

Comment:
Yeah it worked fine. Have you tried building this branch?

Reply:
not yet, that's why I'm asking. I tried with my last vector PR, therefore the comment

Reply:
Okay, the drag'n'drop icon appears as before.
Where there any changes recently that make it work with svg now? Hm..

Reply:
Thank you for testing. Just to double check, ready for merge?

Reply:
DnD works.
You probably need some else to confirm **window icons** show up correctly. @daschuer mentioned issues recently, and I can't test since I don't see ANY window icons with xfce4 (bug? hmm don't care)

Reply:
Ping @daschuer, could you test if the window icon works for you with this branch?